VirusBuster/src/BulletReflect.cpp

```diff
@@ -0,0 +1,29 @@
+﻿#include"ObjectClass.hpp"
+
+BulletReflect::BulletReflect(RectF body, double p_angle) {
+	Body = body;
+	angle = p_angle; //- Math::Pi / 2;
+	reflectCount = 10;
+	damage = 1;
+}
+
+void BulletReflect::move() {
+	static constexpr float speed = 4;
```

Comment:
constexpr使うなら、staticいらないのでは?
```suggestion
	constexpr float speed = 4;
```

Reply:
一応[違い](https://www.webdevqa.jp.net/ja/c%2B%2B/%E3%81%84%E3%81%A4%E3%80%81%E3%81%AA%E3%81%9Cconstexpr%E3%81%A7static%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%99%E3%82%8B%E3%81%AE%E3%81%A7%E3%81%99%E3%81%8B%EF%BC%9F/1049279948/)はあるっぽいのですが、なんだか難しい話なので問題ないと思います。
実際、constexprに変えようと思って怠けただけのミスなので異論はありません。